test/common/README.md

```diff
@@ -329,5 +334,13 @@ Node.js
 implementation with tests from
 [W3C Web Platform Tests](https://github.com/w3c/web-platform-tests).
 
-[MDN-Function]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Normal_objects_and_functions
 [internationalization]: https://github.com/nodejs/node/wiki/Intl
+[&lt;Array>]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
```

Comment:
A nit: if we use an ASCII sorting for references, should not the `[&lt;...` ones precede the `[internationalization]`?